pkgs/by-name/rb/rbenv/package.nix

```diff
@@ -24,19 +29,20 @@ stdenv.mkDerivation rec {
     mv libexec $out
     ln -s $out/libexec/rbenv $out/bin/rbenv
 
-    installShellCompletion completions/rbenv.{bash,zsh}
+    installShellCompletion --zsh completions/_rbenv
+    installShellCompletion --bash completions/rbenv.bash
```

Comment:
what's this change?

Reply:
This change is about 2 things, @fzakaria:
* Upstream renamed `completions/rbenv.zsh` to `completions/_rbenv`
* I felt adding the `--zsh` and `--bash` option would convey more explicitly which type of shell completion is installed.